### PR TITLE
Add a download via QR code

### LINKED ISSUE #2956: Add download via QR code
**Describe the feature you'd like**

Add a QR code image which when scanned would redirect to the Play Store where users could download the Phimp me application.

**Screenshots**

**Additional context**

This could be an easy way for developers or viewers alike to visit and download the Phimp me application, right from GitHub.

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by pranavkarthik10**
Fixed #2956 

Changes:
Added a QR code which enables users to actively access the Play Store where they can download the application. This could be very useful for users on their desktop/laptop who want to download it on their phone.

Screenshots of the change: 
Screen Shot 2019-12-30 at 3 08 48 PM

### CODE REVIEW

**Comment by pranavkarthik10** [comment]
@mariobehling Please review and merge! Thanks :)

File: README.md [hunk 1]
```diff
@@ -11,6 +11,9 @@
 |------------|-----------------|-------------|-----------------|-----------------|-----------------|
 | [![Build Status](https://travis-ci.org/fossasia/phimpme-android.svg?branch=master)](https://travis-ci.org/fossasia/phimpme-android) | [![Build Status](https://travis-ci.org/fossasia/phimpme-android.svg?branch=development)](https://travis-ci.org/fossasia/phimpme-android) | [![codecov](https://codecov.io/gh/fossasia/phimpme-android/branch/master/graph/badge.svg)](https://codecov.io/gh/fossasia/phimpme-android) | [![Codacy Badge](https://api.codacy.com/project/badge/Grade/4584003e734343b3b8ce94bcae6e9ca4)](https://www.codacy.com/app/harshithdwivedi/phimpme-android?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=fossasia/phimpme-android&amp;utm_campaign=Badge_Grade) | [![Gitter Room](https://img.shields.io/badge/gitter-join%20chat%20%E2%86%92-blue.svg)](https://gitter.im/fossasia/phimpme) | [![Twitter Follow](https://img.shields.io/twitter/follow/phimpme.svg?style=social&label=Follow&maxAge=2592000?style=flat-square)](https://twitter.com/phimpme) |
 
+## Download QR Code
```
Comment (yashk2000) [inline README.md]: Instead of saying `Download QR Code`, say ` Download from PlayStore`.

**Comment by pranavkarthik10** [comment]
@yashk2000 Updated with the new heading!

**Comment by pranavkarthik10** [comment]
@yashk2000 @mariobehling Please review this. Thank you very much!

**Pull request merged**